eggdrop.conf

```diff
@@ -615,6 +615,19 @@ die "Please make sure you edit your config file completely."
 set mod-path "modules/"
 
 
+#### PBKDF2 MODULE ####
+#loadmodule pbkdf2
+
+# Encryption method used. openssl list -digest-algorithms
+#set pbkdf2-method "SHA512"
```

Comment:
About the comment, SHA512 isn't an encryption method.

Reply:
I know I'm being pedantic but this whole change is for a salted hash which isn't encryption because it cannot be decrypted.

I support the change in principle, one-way hashing is good. There's no reason a password needs to be decrypted by eggdrop.

Reply:
Yeah, ill change that, thanks!